packages/db/postgres/docker/docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
       PGRST_DB_ANON_ROLE: postgres
       PGRST_JWT_SECRET: super-secret-jwt-token-with-at-least-32-characters-long
       PGRST_LOG_LEVEL: info
+      PGRST_DB_SCHEMAS: public, cargo
```

Comment:
Despite this not being not required, I left it here since it could be useful in the future if we want to query cargo schema directly fom DbClient locally.